### PR TITLE
Handing in my project

### PR DESCRIPTION
**Pull request by SofieFerrari**
Link to my deployed project: https://65b65abf54edd85f37cf095b--sunny-nasturtium-d24122.netlify.app/
Correct link: https://master--sunny-nasturtium-d24122.netlify.app/.

Reflections: I have to settle when I'm done and trust that I'm done so that I don't end up with things like this where i change something and don't have the time to fix it befor I have to hand it in.

Explanation: I did the project twice, I won't to that again. Then my child got hospitalized so I had to rush everything from the hospital, that's why I haven't cleaned up my code and fixed a layout thing. I explained in the README-file!

### CODE REVIEW

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Sofie, well let's start with a well done for working on your project despite all the difficulties! 💪 

I'll need to ask you to apply these changes to fulfill this assignment:
- The app needs to be responsive with the following views: Desktop 4 columns, tablet 2 columns, mobile 1 column (it should look good on devices from 320px width up to 1600px)

Another important improvement is to create a folder for the images, it's good practice to separate in groups the files in a project. Make sure to apply these improvements to get your Pull Request approved 🙌 Thank you :)

**Comment by AntonellaMorittu** [comment]
Hi Sofie, the app should display 4 columns of articles on desktop, 2 on tablet and 1 column on mobile. Right now you have all of your items displayed in 2 rows? I recommend to try follow the codealong about grids we have in our material [here](https://www.notion.so/technigo/Pre-Course-Tools-HTML-CSS-Flexbox-Grid-0ed0bd0101db40a2b36e61836a45c5d5?pvs=4#db3e3004003244c7914895e5a8d50b15) to help you practice :)

**Comment by SofieFerrari** [comment]
Hi! I think you clicked the wrong link, the right one is i my readme-file: https://master--sunny-nasturtium-d24122.netlify.app/. 
This is what I did today and it displays 1 col for mobile, 2 for tablet and 4 for desktop! 

I'll try to delete the wrong link! Hope you have a nice weekend!

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,161 @@
...
+    justify-content: flex-start;
+    color: rgb(193, 85, 139);
+    font-size: 60px;
+    margin: 5px;
+    padding: 5px;
+  }
+
+  .image-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax (200px, 1fr));
```
Comment (AntonellaMorittu) [inline code/style.css:160]: same here, you want to display 4 columns on desktop, therefore you need to use a rule that explicitly does it, like `grid-template-columns: repeat(4, 1fr)` for example

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Sogie, almost there! Check my comments and update your project accordingly. Thank you 💪

File: code/style.css (hunk 1 above)
Comment (SofieFerrari) [reply code/style.css]: Thank you Antonella! I really hope I fixed it now!

**Review by AntonellaMorittu — APPROVED** [review]

**Pull request closed without merging**